apps/vs-code-react/src/app/overview/app.tsx

```diff
@@ -1,69 +1,43 @@
-// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-import messages from '../../../../../libs/services/intl/src/compiled-lang/strings.json';
 import { QueryKeys } from '../../run-service';
 import type { RunDisplayItem } from '../../run-service';
-import type { OnErrorFn } from '@formatjs/intl';
+import type { RootState } from '../../state/store';
+import { VSCodeContext } from '../../webviewCommunication';
 import { StandardRunService } from '@microsoft/designer-client-services-logic-apps';
 import type { CallbackInfo } from '@microsoft/designer-client-services-logic-apps';
-import type { OverviewPropertiesProps } from '@microsoft/designer-ui';
 import { Overview, isRunError, mapToRunItem } from '@microsoft/designer-ui';
 import type { Runs } from '@microsoft/utils-logic-apps';
-import { HttpClient } from '@microsoft/vscode-extension';
-import { useCallback, useMemo } from 'react';
-import { IntlProvider } from 'react-intl';
-import { QueryClient, QueryClientProvider, useInfiniteQuery, useMutation } from 'react-query';
+import { ExtensionCommand, HttpClient } from '@microsoft/vscode-extension';
+import { useCallback, useContext, useMemo } from 'react';
+import { useInfiniteQuery, useMutation } from 'react-query';
+import { useSelector } from 'react-redux';
 import invariant from 'tiny-invariant';
 
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      refetchInterval: false,
-      refetchIntervalInBackground: false,
-      refetchOnWindowFocus: false,
-      refetchOnReconnect: false,
-      refetchOnMount: true,
-    },
-  },
-});
-
-export interface AppProps {
-  apiVersion: string;
-  baseUrl: string;
-  workflowProperties: OverviewPropertiesProps;
-  corsNotice?: string;
-  accessToken?: string;
-  onOpenRun(run: RunDisplayItem): void;
-  hostVersion?: string;
-}
+export const OverviewApp = () => {
+  const workflowState = useSelector((state: RootState) => state.workflow);
+  const vscode = useContext(VSCodeContext);
 
-export const App: React.FC<AppProps> = (props) => {
```

Comment:
We are already checking this initialized part in stateWrapper; we only navigate to this part if it is initialized. Removing duplicate QueryClientProvider & IntlProvider, we don't need another layer in index.tsx. Removed index.tsx and directly used workflowState to refer to values